Scripts/developer_scripts/cgal_create_release_with_cmake.cmake

```
@@ -11,7 +11,7 @@
 # GENERATE_TARBALLS=[ON/OFF] indicates if release tarballs should be created as DESTINATION
 
 cmake_minimum_required(VERSION 3.1...3.15)
-
+find_program(BASH bash sh)
```

```suggestion
find_program(BASH NAMES bash sh)
```
See https://cmake.org/cmake/help/latest/command/find_program.html

With your code, `find_program` searches only for the name `bash`, and it searches in sub-directories `sh/` of several prefixes.